### ISSUE TITLE
Any effect on Google Analytics 

### ISSUE THREAD

**Issue by jaydeep17** [romainbutteaud/Kaffeine#8]
Hey,

I have Google analytics setup on my webpage, will this ping requests show up there ?

P.S. Really handy service :+1: 

Thanks.

**Comment by romainbutteaud** [comment]
@jaydeep17 I believe it will yes

**Comment by matijaabicic** [comment]
i don't think it will. If the piece of GA javascript isn't executed, you won't get a hit in google analytics.Since Kaffeine probably isn't executing javascript (i'm guessing you're issuing a simple get request), you won't see a ping from Kaffeine. You can test this with curl or wget on a low-traffic site.
